.github/workflows/main.yml

```diff
@@ -0,0 +1,103 @@
+name: Main CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        version: [max] # [x.x.x | latest | max]
+        type: [stable] # [stable | insider]
+      fail-fast: false
+
+    env:
+      CODE_VERSION: ${{ matrix.version }}
+      CODE_TYPE: ${{ matrix.type }}
+      VSCODE_CAMELK_GITHUB_TOKEN: ${{ github.token }}
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: npm
+
+      - name: Install global dependencies
+        run: npm install -g typescript @vscode/vsce
+  
+      - name: npm-ci
+        run: npm ci
+  
+      - name: npm-compile
+        run: npm run compile
+
+      - name: Start minikube
+        uses: medyagh/setup-minikube@v0.0.11
+        with:
+          minikube-version: 1.29.0
```

Comment:
I updated to 1.30.1on Circle Ci config. Woudl be nice to align it.

Reply:
Done, also bumped medyagh/setup-minikube@v0.0.11 to medyagh/setup-minikube@v0.0.13